docs/source/entities-advanced.mdx

```diff
@@ -70,8 +67,7 @@ Let's say our Payments subgraph defines a `Bill` entity:
 
 <CodeColumns>
 
-```graphql
-# Payments subgraph
+```graphql title="Payments subgraph"
```

Comment:
Does this title do what you're looking for? If so, there's a million of these changes so I'm good if that answer is "yes".

Reply:
It very much does! 😄